features/book_tickets.feature

```diff
@@ -21,6 +21,13 @@ Feature: Book tickets
     And I click on an event
     And I click Book Now button
     Then I should be at the Booking page
+    And I can see a list of ticket types order by price desc
+    When I click "Buy"
+    Then I can see "Please choose at least 1 ticket to continue!"
+    When I select the quantity of tickets
+    And I click "Buy"
+    Then I should be redirected to the order's details page
+    And I can see "Order successfully!"
```

Comment:
`And I can see the order's total sum` maybe, just to be sure that you are being redirected to the order you just created.